docs/nodes/credentials/Microsoft/README.md

```diff
@@ -0,0 +1,22 @@
+# Microsoft
+
+You can find information about the operations supported by the OneDrive node on the [integrations](https://n8n.io/integrations/n8n-nodes-base.microsoftOneDrive) page. You can also browse the source code of Microsoft nodes on [GitHub](https://github.com/n8n-io/n8n/tree/master/packages/nodes-base/nodes/Microsoft).
```

Comment:
Why is OneDrive specifically pointed out here? Perhaps better to have a table like in AWS.